sdk/metricsadvisor/azure-ai-metricsadvisor/README.md

```diff
@@ -172,6 +176,9 @@ for status in ingestion_status:
 ```
 
 ### Configure anomaly detection configuration
+
+We need an anomaly detection configuration to determine whether a point in the time series is an anomaly.
```

Comment:
There's a default one though.  We need to create if we want customization.

Reply:
How about adding the following? "While a default detection configuration is automatically applied to each metric, you can tune the detection modes used on your data by creating a customized anomaly detection configuration"